contracts/FindMarketOptions.cdc

```diff
@@ -138,7 +138,7 @@ pub contract FindMarketOptions {
                 }
 
                 if let validTime = item.getValidUntil() {
-                    if validTime >= Clock.time() {
+                    if validTime <= Clock.time() {
```

Comment:
Nice catch, I hope we have a test for this now?

Reply:
Yes, test is already added for this. 
The script should return "active" listings.